ang/civicase/Dashboard.html

```diff
@@ -0,0 +1,149 @@
+<div id="bootstrap-theme" class="civicase__container">
+  <div class="civicase__dashboard">
+    <h1 class="hidden" crm-page-title crm-document-title="myCasesOnly ? ts('CiviCase Dashboard - My Cases') : ts('CiviCase Dashboard - All Cases')"></h1>
+
+    <div class="civicase__dashboard__tab-container clearfix">
+      <uib-tabset
+        civicase-dashboard-tabset-affix active="activeTab" type="pills"
+        uib-tabset-class="nav-pills-horizontal nav-pills-horizontal-primary clearfix">
+        <!-- Dashboard Tab -->
+        <uib-tab index="0" heading="{{ ts('Dashboard') }}" select="navTabs()">
+
+          <div class="panel panel-default" ng-include="'~/civicase/DashboardSummary.html'"></div>
+
+          <div class="row case-stats-wrapper">
```

Comment:
`case-stats-wrapper` need to follow BEM.

Reply:
The whole file has been renamed, thats why everything is showing as changed.
But this HTML is part of Dashboard tab, which is not touched in this PR.
It needs to be taken care of when we work on the Dashboard tab ticket.